static/js/popovers.js

```diff
@@ -234,6 +237,21 @@ function show_user_profile(element, user) {
     $("#user-profile-modal-holder").html(templates.render("user_profile_modal", args));
     ui.set_up_scrollbar($("#user-profile-modal-body"));
     $("#user-profile-modal").modal("show");
+
+    page_params.custom_profile_fields.forEach(function (field) {
+        var field_value = people.get_custom_profile_data(user.user_id, field.id);
+
+        if (field.type === field_types.USER.id && field_value) {
+            field_value = JSON.parse(field_value);
```

Comment:
This sort of type rewriting: ` field_value = JSON.parse(field_value);` is bad style; I changed the initial variable to be `var field_value_raw`